script.bluetooth.delay/addon.py

```diff
@@ -34,89 +39,70 @@ def skin2():
     xbmc.executebuiltin("Action(select)")
     xbmc.executebuiltin('SetFocus(11)')
     xbmc.executebuiltin("Action(select)", wait=True)
-    xbmc.executebuiltin("Action(close)", wait=True)
-
-# reset for first run kodi
-def reset1():
-    xbmc.executebuiltin('ActivateWindow(osdaudiosettings)')
-    xbmc.executebuiltin('SetFocus(-77)')
-    xbmc.executebuiltin("Action(select)")
-    xbmc.executebuiltin('SetFocus(11)')
-    xbmc.executebuiltin("Action(select)", wait=True)
-    xbmc.executebuiltin("Action(close)", wait=True)
-
-def reset2():
-    xbmc.executebuiltin('ActivateWindow(osdaudiosettings)')
-    xbmc.executebuiltin('SetFocus(-78)')
-    xbmc.executebuiltin("Action(select)")
-    xbmc.executebuiltin('SetFocus(11)')
-    xbmc.executebuiltin("Action(select)", wait=True)
+    xbmc.sleep(1)
     xbmc.executebuiltin("Action(close)", wait=True)
 
 if "ace2" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "aeon.nox.silvo" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "aeon.tajo" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "aeonmq8" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "ftv" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "madnox" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "pellucid" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "quartz" in xbmc.getSkinDir():
      skin1 = skin2
-     reset1 = reset2
 elif "xperience1080" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 elif "mimic.lr" in xbmc.getSkinDir():
     skin1 = skin2
-    reset1 = reset2
 
 def bluetooth():
     Addon.setSettingBool('state', 0)
     for x in range(y):
         xbmc.executebuiltin("Action(AudioDelayPlus)")
-    xbmcgui.Dialog().notification("",line2, "",t)
+    xbmcgui.Dialog().notification(d2,line2,icon,t)
 
 def speakers():
     Addon.setSettingBool('state', 1)
     for x in range(y):
         xbmc.executebuiltin("Action(AudioDelayMinus)")
-    xbmcgui.Dialog().notification("",line1, "",t)
+    xbmcgui.Dialog().notification(d1,line1,"",t)
 
 def main():
 
     if arg == "reset":
-        xbmc.executebuiltin('PlayerControl(stop)')
-        Addon.setSettingBool('state', 1)
-        reset1()
-        skin1()
-        xbmcgui.Dialog().notification("",Addon.getLocalizedString(30006), "",t*2)
+        if not xbmc.getCondVisibility('Player.HasVideo'):
+            xbmcgui.Dialog().notification("",Addon.getLocalizedString(30006), "",t)
+        else:
+            if firstRun == "false":
+                if xbmcgui.Dialog().ok(Addon.getLocalizedString(30007), Addon.getLocalizedString(30008)) == True:
```

Comment:
`is True`

Reply:
Do you mean `if xbmcgui.Dialog().ok(Addon.getLocalizedString(30007), Addon.getLocalizedString(30008)) is True:` ?
Because it does not work properly.